tests/floats/.gitignore

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
Please remove this file.

Reply:
Sure!

Reply:
Does this require anything else to be done?

Reply:
Yes. CI fails.